data_steward/cdr_cleaner/cleaning_rules/deid/remove_fitbit_data_if_max_age_exceeded.py

```diff
@@ -135,12 +140,16 @@ def validate_rule(self, client):
         """
         raise NotImplementedError("Please fix me.")
 
+    def sandbox_table_for(self, affected_table):
```

Comment:
We can inherit `sandbox_table_for` from the base class and use it. So, we do not need to define it here.